doc/auto-discovery-docker.md

```diff
@@ -29,11 +29,11 @@ services:
     network_mode: "host"
     environment:
       SERVICE_HOST: 192.168.56.101
+      SERVICE_SERVERBINDADDR: 0.0.0.0
```

Comment:
I thought we are getting rid of this and documenting separately how to configure to the ONVIF rest callback notifications. i.e we don't want to open this up by default, only when the user wants ONVIF rest callback and knows the risks of opening it up like this.

So for now this should be removed and service `SERVICE_HOST: edgex-device-onvif-camera` and  remove `network_mode: "host"`

Reply:
Yes, I was a little confused as to why this code was showing as being new/different, as it was originally changed in main branch. I think maybe a rebase caused to to be seen as a different commit. We can remove it.